src/execution_plan/ops/op_expand_into.c

```
@@ -223,11 +223,15 @@ static Record _handoff
 		NodeID col      =  ENTITY_GET_ID(destNode);
-		// TODO: in the case of multiple operands ()-[:A]->()-[:B]->()
-		// M is the result of F*A*B, in which case we can switch from
-		// M being a Delta_Matrix to a GrB_Matrix, making the extract element
-		// operation a bit cheaper to compute
-		GrB_Info res    =  Delta_Matrix_extractElement_BOOL(&x, op->M, row, col);
+		GrB_Info res;
+		if(op->ae->type == AL_OPERATION) {
+			GrB_Matrix M = Delta_Matrix_M(op->M);
+			res = GrB_Matrix_extractElement_BOOL(&x, M, row, col);
+		} else if(op->ae->operand.transpose) {
+			res = Delta_Matrix_extractElement_BOOL(&x, op->M, col, row);
```

cppcheck

[error] Common realloc mistake

(memleakOnRealloc)